draft-ietf-rtcweb-jsep.xml

```diff
@@ -3199,12 +3212,20 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               the size limits specified in
               <xref target="RFC5245" />, Section 15.4.</t>
 
+              <t>dtls-id value, which MUST be set according to
+              <xref target="I-D.ietf-mmusic-dtls-sdp"/> Section 5.
+              If this is a re-offer and the dtls-id value is different
+              from that presently in use, then a new DTLS connection
+              MUST be established, and the ufrag and password values
+              MUST also be different. If processing an answer, the
+              dtls-id value, if present, MUST be the same as in the
+              offer.</t>
+
               <t>DTLS setup value, which MUST be set according to the
-              rules specified in
-              <xref target="RFC5763" />, Section 5, and MUST be
-              consistent with the selected role of the current DTLS
-              connection, if one exists.[TODO: may need revision, i.e.,
-              use of actpass</t>
+              rules specified in <xref target="RFC5763"/>,
+              Section 5 and MUST
+              be consistent with the selected role of the current DTLS
+              connection, if one exists and is being continued.</t>
```

Comment:
I also think you need some text in the "applying a remote description" section below that talks about creating a new DTLS connection.